package.json

```diff
@@ -282,7 +282,7 @@
     "@metamask/address-book-controller": "^4.0.1",
     "@metamask/announcement-controller": "^6.1.0",
     "@metamask/approval-controller": "^6.0.1",
-    "@metamask/assets-controllers": "^29.0.0",
+    "@metamask/assets-controllers": "^30.0.0",
```

Comment:
This will require other changes, as you found form https://github.com/MetaMask/metamask-extension/pull/24913

Reply:
yes indeed this PR is blocked by the assets-controllers bump being currently worked on #24913